pkg/crc/ssh/ssh.go

```diff
@@ -90,7 +90,7 @@ func (runner *Runner) runSSHCommand(command string, runPrivate bool) (string, st
 	if err != nil {
 		return string(stdout), string(stderr), fmt.Errorf(`ssh command error:
 command : %s
-err     : %w\n`, command, err)
+err     : %w`+"\n", command, err)
```

Comment:
or just add a carriage return since this is already a multi-line string?
```
fmt.Errorf(`ssh command error:
command : %s
err     : %w
`, command, err)
```

Maybe the short log can be reworded to `ssh: Append new line instead of '\n' to error string` ?

Reply:
Sure we can do that.